explainer.md

```diff
@@ -40,17 +39,17 @@ Possible areas for expansion:
   proposal is just for installed apps (designed to show up in the operating
   system shelf area). We could also explore icon badging on the drive-by web.
   This naturally leads into...
-* Provide per-window badging information. The current proposal is for a global
-  badge for the application, not per-window or per-tab. See
-  [#1](https://github.com/WICG/badging/issues/1).
+* Providing per-tab and per-window badging. The current proposal
+  is for an application badge See [#1](https://github.com/WICG/badging/issues/1).
```

Comment:
Missing a period. Also why did this sentence change from "The current proposal is for a global badge for the application" to "The current proposal is for an application badge"? That seems less clear to me.